Documentation/op-guide/recovery.md

```diff
@@ -61,3 +61,25 @@ $ etcd \
 ```
 
 Now the restored etcd cluster should be available and serving the keyspace given by the snapshot.
+
+## Restoring a cluster from membership mis-reconfiguration
+
+Previously, etcd panics on [membership mis-reconfiguration with wrong URLs](https://github.com/coreos/etcd/issues/9173). v3.2.15 and v3.3.0+ return [error early in client-side](https://github.com/coreos/etcd/pull/9174) before etcd server panic.
+
+To fix such misconfiguration while keeping original data, `--force-new-cluster` flag can be used to overwrite cluster configuration. Please be CAUTIOUS when using this flag because it will panic if other members from previous cluster are still alive. Please follow the instructions below.
```

Comment:
i do not think we should promote the `--force-new-cluster` command anymore. we should instead promote the restore from snapshot way.

Reply:
Ah, that was why it's only documented in v2 docs. Closing.